tensorflow/compiler/mlir/README.md

```diff
@@ -19,27 +19,17 @@ Building dialects and utilities here follow the standard approach using
 
 To develop across MLIR core and TensorFlow, it is useful to override the repo
 to use a local version instead of fetching from head. This can be achieved by
-setting up your local repository for Bazel build. For this you will need a
-temporary directory that will be "overlaid" with you LLVM source directory and
-the Bazel files:
+setting up your local repository for Bazel build. For this you will need to
+create bazel workspace and build files:
 
 ```sh
 LLVM_SRC=... # this the path to the LLVM local source directory you intend to use.
-LLVM_BAZEL_OVERLAY=${LLVM_SRC}/bazel # Note: this can be anywhere
-mkdir -p ${LLVM_BAZEL_OVERLAY}
-# This will symlink your LLVM sources with the BUILD files to be usable by Bazel.
-python ${LLVM_SRC}/utils/bazel/overlay_directories.py \
-    --src ${LLVM_SRC} \
-    --overlay ${LLVM_SRC}/utils/bazel/llvm-project-overlay/ \
-    --target ${LLVM_BAZEL_OVERLAY}
-touch ${LLVM_BAZEL_OVERLAY}/BUILD.bazel ${LLVM_BAZEL_OVERLAY}/WORKSPACE
-# The complete list is "AArch64", "AMDGPU", "ARM", "NVPTX", "PowerPC", "RISCV", "SystemZ", "X86"
-echo 'llvm_targets = ["X86"]' > ${LLVM_BAZEL_OVERLAY}/llvm/targets.bzl
+touch ${LLVM_SRC}/BUILD.bazel ${LLVM_SRC}/WORKSPACE
 ```
 
 You can then use this overlay to build TensorFlow:
 
 ```
-bazel build --override_repository=llvm-project=$LLVM_BAZEL_OVERLAY \
+bazel build --override_repository=llvm-raw=$LLVM_BAZEL_OVERLAY \
   -c opt tensorflow/compiler/mlir:tf-opt
```

Comment:
The instructions no longer define `$LLVM_BAZEL_OVERLAY`. Should it be `$LLVM_SRC`?

Reply:
Oh good catch, yeah it should